src/schema/v2/__tests__/tag.test.js

```diff
@@ -35,9 +36,9 @@ describe("Tag", () => {
       const query = `
         {
           tag(id: "butt") {
-            filtered_artworks(aggregations:[TOTAL]){
+            filteredArtworksConnection(aggregations:[TOTAL]){
```

Comment:
This doesn't have any `hits`, which was a non-connection list. Since `filterArtworksConnection` is the actual connection, it has edges, etc. I wonder if replacing `hits` with the `edges { node { slug } }` would work.